src/Service/main.cpp

```diff
@@ -103,6 +103,7 @@ int main(int argc, char** argv) {
 
   if (!result.has_error()) return 0;
 
+  ORBIT_ERROR("%s", result.error().message());
```

Comment:
So all the `ORBIT_ERROR`s that have been turned into `ErrorMessage` are printed here now?

Reply:
Yes. Does that make sense to you?

Reply:
Yep, just confirming.